data/remnant ships.txt

```diff
@@ -583,3 +718,57 @@ ship "Starling" "Starling (Thrasher)"
 	gun
 
 
+
+ship "Tern"
+	sprite "ship/tern"
+	thumbnail "thumbnail/tern"
+	attributes
+		category "Drone"
+		licenses
+			Remnant
+		"cost" 860000
+		"shields" 500
+		"hull" 2500
+		"automaton" 1
+		"cargo space" 5
+		"mass" 32
+		"drag" 1.0
+		"heat dissipation" .9
+		"fuel capacity" 50
+		"energy capacity" 500
+		"ramscoop" 0.5
+		"outfit space" 75
+		"weapon capacity" 16
+		"engine capacity" 26
+		"shield generation" 0.3
+		"shield energy" 0.2
+		"hull repair rate" 0.6
+		"hull energy" 0.4
+		"gaslining" 1
+		"atmosphere scan" 100
+		"outfit scan power" 12
+		"outfit scan speed" 1
+		"tactical scan power" 26
+		"asteroid scan power" 30
+		weapon
+			"blast radius" 10
+			"shield damage" 100
+			"hull damage" 150
+			"hit force" 250
+	outfits
+		"Inhibitor Cannon"
+		
+		"Millennium Cell"
+		"Thermoelectric Cooler"
+		
+		"Anvil-Class Engine"
+		
+	gun 0 -22 "Inhibitor Cannon"
+	engine -5 20.5
+	engine 5 20.5
+	explode "tiny explosion" 30
+	explode "small explosion" 20
+	description "The Tern is a small, hardy drone developed to plunge into the atmosphere of nearby gas giants for collecting valuable resources. These thick-shelled craft are also surprisingly effective when pressed into service providing distraction and covering fire when enemies intrude on Remnant space."
+	description "	Unlike most empires, the Remnant do not have sufficient people to pilot fighter wings in waves likely to sustain significant casualties. As such, they frequently use these hardy drones to fulfill similar screening roles in their home systems, allowing manned craft to focus on more specialized tasks."
```

Comment:
sufficient people -> sufficient amounts of people ?

Reply:
changed "sufficient" to "enough," as that sounds a bit better.